checkov/cloudformation/graph_builder/local_graph.py

```diff
@@ -232,6 +252,16 @@ def _find_fn_sub_parameter(string):
         regex = re.compile(r'\${([a-zA-Z0-9.]*)}')
         return regex.findall(string)
 
+    def _fill_in_out_edges(self) -> None:
+        for i, vertex in enumerate(self.vertices):
+            if i not in self.in_edges:
+                self.in_edges[i] = []
+            if i not in self.out_edges:
+                self.out_edges[i] = []
```

Comment:
Not sure what you meant? At this point `self.in_edges` and `self.out_edges` are already assigned with dicts, I'm filling the existing dicts wit empty lists at the i'th key

Reply:
@Saarett This is the initialization of `in_edges and out_edges`. Meaning there is now need to do this init again, since using `defaultdict(list)` allows setting an empty list for each key when inserting it.
```
self.in_edges: Dict[int, List[Edge]] = defaultdict(list)  # map between vertex index and the edges entering it
self.out_edges: Dict[int, List[Edge]] = defaultdict(list)  # map between vertex index and the edges exiting it
```

Reply:
The problem im trying to deal with this function is enumeration of existing keys, not getting non-exisitng keys with self.in_edges[i]. For reference, please look at `get_vertices_with_degrees_conditions` of CloudformationLocalGraph